### ISSUE TITLE
Dataview for filtering Tasks from Pages that have a key::value on them

### ISSUE THREAD

**Issue by ArneNostitz** [roovo/obsidian-card-board#123]
It would be great to have a filter that looks for tasks in files that have a key::value pair on them. I have an extensive system to "tag" files that belong to a project with "Project:: Name". I dont use tags for this because i want everything connected as a backlink to my project dashboard. and using this method for a year now.
Filtering out tasks with the tasks plugin or the dataview query works fine like that - so i dont have to #project every single task. 

any way to do this now? or maybe implement that? thanks!

**Comment by roovo** [comment]
Hiya and thank you.  The only suggestion I can make with the current way that CardBoard works is (assuming that you have your tasks in files that belong to a specific project) to put a project tag in the front matter, as any tags in the front matter will be applied to all tasks in the document.  This may help to get you what you need (hopefully).

Outside of this though, if I understand correctly, what you would ideally like is for CardBoard to support using Dataview "tags" to build boards as an alternative to using regular #tags.  Interesting idea which I have not given any thought to as I am only a very light user of Dataview.

**Comment by roovo** [comment]
(note to self) also see #51

**Comment by ArneNostitz** [comment]
yes exactly. the Frontmatter in most files does have a "project: name" yaml 
i would need to find a tool to auto tag all old and new files. but that's a good workaround. thanks! 

(i do use kanban for now with more manual process. good thing here is i can "send" tasks to an inbox and drag them to various columns based on topic. this does not tag them on the source though. so semi perfect. also a workaround ;-) )

**Comment by ArneNostitz** [comment]
this could also be great later on to attach tasks to project members and sync over to asana or such a thing

**Comment by ArneNostitz** [comment]
getting off track. sorry.

**Comment by roovo** [comment]
Fab - thanks for getting back.  I think I know what would be needed for this.  Will leave it labelled as it is for now and have a fuller think and backlog prioritization after I have got drag and drop out of the way (which will take some time I think).

On the assign tasks to people front - I am pondering this.  I'd love to have it so you'd get a little picture of the person (perhaps I should integrate with gravatar) on a card or something - but that's something for a separate issue at some point :)  At the moment I am using #people/name tags in my vaults.  There are some things that aren't particularly satisfactory about that approach at the moment tho - specially when tagging sub-tasks for people; the bit I'm working on at the moment is to (hopefully) improve this somewhat.

Thank you again.  I will keep this issue updated as I have thoughts and when I get closer to being able to look into it properly.  All the best for 2023 :)

**Comment by benhamilton** [comment]
>At the moment I am using #people/name tags in my vaults
I've create a page for each person (`/Resources/_person/GivenName LastName.md`) for each person and organisation (`/Resources/_organisation/Organisation Name.md`) that I mention in tasks or documentation, and that way my tasks look like this:
- [ ] [[Acme Group]] awaiting response from [[Wile E Coyote]] regarding [[Integration of X with Y]] #pending  

This means that I can go to that persons page and see all the linked references, and thus see what I'm waiting on from them.

**Comment by ArneNostitz** [comment]
that is a very cool system. i use something quite similar for single tasks. but most of the time it's a meeting note with the name of the person as person::name somewhere on the page .

tasks are then attached to that meeting. so i have references to all needed info (when summarizing the tasks i get the link to the file. the file has the name of the person in the filename too)

but maybe a collaboration with the programmer of the tasks plugins would be nice. it could introduce the possibility for variable new fields (next to due and start,..). best self configurable. so it would be possible to set a "people emoji" (tasks) for people key (Dataview,cardboard) for example. or a "connection" emoji for "connected task". or....

**Comment by AlmuthMueller** [comment]
> Fab - thanks for getting back. I think I know what would be needed for this. Will leave it labelled as it is for now and have a fuller think and backlog prioritization after I have got drag and drop out of the way (which will take some time I think).
> 
> On the assign tasks to people front - I am pondering this. I'd love to have it so you'd get a little picture of the person (perhaps I should integrate with gravatar) on a card or something - but that's something for a separate issue at some point :) At the moment I am using #people/name tags in my vaults. There are some things that aren't particularly satisfactory about that approach at the moment tho - specially when tagging sub-tasks for people; the bit I'm working on at the moment is to (hopefully) improve this somewhat.
> 
> Thank you again. I will keep this issue updated as I have thoughts and when I get closer to being able to look into it properly. All the best for 2023 :)

Heya, I use another variation for a workflow for tasks that are assigend to other people and projects. I also use inline dataview fields like [assigned:: [[Person]]] or [in_project:: [[Projectpage]]. 

So when implenenting dataview handling in cardboard it would be really greate if it is possible to use those dataview fields to create boards for people and projects and eg. have a column in the project board for each person working on that project, etc.

And thanks for the great plugin, it helps me a lot!
